### PR TITLE
NODE-2634/ocsp-4.4-latest-only

### PR DESCRIPTION
**Pull request by reggi**
https://jira.mongodb.org/browse/NODE-2634?filter=-1

### CODE REVIEW

**Comment by reggi** [comment]
I couldn't get `aws-auth-test` to use the VERSION argument, so I reverted it. I'm not sure how to run tests for `4.4` and `latest` for it.

File: .evergreen/generate_evergreen_tasks.js [hunk 1]
```diff
@@ -100,137 +101,140 @@ MONGODB_VERSIONS.forEach(mongoVersion => {
...
+        },
+        { func: 'run-ocsp-test', vars: { OCSP_TLS_SHOULD_SUCCEED: 0 } }
+      ]
+    }
+  ]);
+})
 
 TASKS.push({
-  name: 'aws-auth-test',
+  name: `aws-auth-test`,
```
Comment (emadum) [inline .evergreen/generate_evergreen_tasks.js:237]: Nitpick but this change to backticks seems unnecessary.

**Comment by mbroadst** [comment]
@reggi are we reviewing this, or waiting for you to get the MONGODB-AWS testing against `latest` complete?

**Comment by reggi** [comment]
This PR only accomplishes part of the ticket, to test OCSP 4.4 and latest, not the AWS auth testing.

**Pull request merged**